docs/server/source/production-deployment-template/easy-rsa.rst

```diff
@@ -69,14 +69,17 @@ The comments in the file explain what each of the variables mean.
    echo 'set_var EASYRSA_REQ_OU "IT"' >> vars
    echo 'set_var EASYRSA_REQ_EMAIL "dev@bigchaindb.com"' >> vars
 
+We follow the convention of modifying the OU to ``ROOT-CA``,
+``MongoDB-Instance``, ``BigchainDB-Instance``, ``MongoDB-Mon-Instance`` and
+``MongoDB-Backup-Instance`` while issuing certificates.
```

Comment:
The OUs need to be different because all the cluster members are in one OU.
This is how MongoDB can differentiate if one of the connections to it is from another cluster member or another client.